README.md

```diff
@@ -6,6 +6,11 @@ web shop.
 
 == Change log ==
 
+** Version 2.0.4
+
+    * Improvements:
+            - Added integration test
```

Comment:
This isn't really a user feature so no point in including in the release notes, or releasing a version where it is the only change.